packages/drivers/odsp-driver/src/contracts.ts

```diff
@@ -202,6 +202,13 @@ export interface ISnapshotOptions {
      * if snapshot is bigger in size than specified limit.
      */
     mds?: number;
+
+    /*
+     * Maximum time limit to fetch snapshot (in seconds)
+     * If specified, client will timeout the fetch request if it exceeds the time limit and
+     * will try to fetch the snapshot without blobs.
```

Comment:
I like the way you described it, but what if host already is asking for snapshot without blobs?
I feel like there should be no fall over in such case (i.e. if it fails, it fails)

Reply:
But in trees latest we always fetch with blobs.

Reply:
But I think I can remove from description here because that will be driver internal behavior?

Reply:
Not really, if the host says no blobs, then we do no blobs. At least that's the case today in the product - this allows host to optimize behavior...